### PR TITLE
Add Superquadric primitives and actors

### PR DESCRIPTION
**Pull request by skoudoro**
Adding the superquadrics actor and primitive. Below a small example to use it

```
from fury import window, actor
scene = window.Scene()
centers = np.random.rand(3, 3) * 10
directions = np.random.rand(3, 3)
colors = np.array([[255, 0, 0], [0, 255, 0], [0, 0, 255]])
scale = np.random.rand(5)
roundness = np.array([[1, 1], [1, 2], [2, 1]])

sq_actor = actor.superquadric(centers, roundness=roundness,
                              directions=directions,
                              colors=colors, scale=scale)
scene.add(sq_actor)
window.show(scene)
```

Capture d’écran 2019-12-06 à 06 47 32

### CODE REVIEW

**Comment by skoudoro** [comment]
should fix #81

File: fury/primitive.py [hunk 1]
```diff
@@ -134,3 +274,47 @@ def prim_sphere(name='symmetric362', gen_faces=False):
...
+    True
+    >>> faces.shape == (720, 3)
+    True
+
+    """
+    def _fexp(x, p):
+        """Return a different kind of exponentiation."""
+        return np.sign(x) * (np.abs(x) ** p)
+
+    sphere_verts, sphere_triangles = prim_sphere('symmetric362')
```
Comment (Garyfallidis) [inline fury/primitive.py]: other spheres...

Comment (skoudoro) [reply fury/primitive.py]: updated!

**Comment by Garyfallidis** [comment]
Cool!

**Pull request merged**